.github/workflows/nix-action-default.yml

```diff
@@ -307,72 +312,6 @@ jobs:
       name: Building/fetching current CI target
       run: NIXPKGS_ALLOW_UNFREE=1 nix-build --no-out-link --argstr bundle "default"
         --argstr job "hydra-battles-single"
-  build-doc:
```

Comment:
Should not be merged yet because I've accidentally committed this removal?

Reply:
I'll leave the merging of the PR to you then....